.github/workflows/build_rpms.yml

```diff
@@ -25,24 +25,44 @@ jobs:
           echo "fastestmirror=1" >> /etc/dnf/dnf.conf
           echo "install_weak_deps=0" >> /etc/dnf/dnf.conf
           dnf -y upgrade
-          dnf -y install dnf-plugins-core git make rpm-build
-          mkdir rpms
+          dnf -y install dnf-plugins-core rpm-build rpmdevtools
 
-      - name: Clone repository
-        uses: actions/checkout@v2
-        with:
-          path: osbuild
+      - name: Fetch .spec file
+        run: |
+          mkdir spec
+          # Fetch the spec file at the given commit, and prepend the commit sha to it
+          curl "https://raw.githubusercontent.com/osbuild/osbuild/${GITHUB_SHA}/osbuild.spec" -o osbuild.spec
+          echo "%global commit ${GITHUB_SHA}" | cat - osbuild.spec > spec/osbuild.spec
```

Comment:
Why not clone the repository and copy the spec-file out? This way you don't have to rely on the URLs and `curl`, and you use the well-defined behavior of github on which sha to fetch, etc..

Reply:
No strong reason, but my thinking was:
 - avoid installing git and it's dependencies in the buildroot (ideally we only install the 
buildd deps of our spec file)
 - avoid the temptation of using other things from the repo

Reply:
Fair enough.